switchapi/es2k/switch_pd_p4_name_mapping.h

```diff
@@ -135,7 +135,7 @@
 #define LNW_L2_FWD_TX_TABLE_KEY_BRIDGE_ID "user_meta.pmeta.bridge_id"
 #define LNW_L2_FWD_TX_TABLE_KEY_DST_MAC "dst_mac"
 
-#define LNW_L2_FWD_TX_TABLE_ACTION_L2_FWD "linux_networking_control.l2_fwd"
+#define LNW_L2_FWD_TX_TABLE_ACTION_L2_FWD "linux_networking_control.l2_fwd_tx"
```

Comment:
Danger, Will Robinson! Non-obvious breaking change between V2 and V3!

Reply:
This was changed for debug purposes, will change it back to l2_fwd. Thanks for pointing it out